scripts/irods/test/test_iquest.py

```diff
@@ -824,8 +825,8 @@ def test_invalid_operations(self):
                 for vop in valid_operations:
                     # First, with the invalid operation as the first clause of the ||...
                     query = f'{self.base_query} {iop} \'{pattern}\' || {vop} \'{pattern}\''
-                    self.user.assert_icommand(['iquest', '%s', query], 'STDERR', 'CAT_INVALID_ARGUMENT')
+                    self.user.assert_icommand(['iquest', '%s', query], 'STDERR', 'INPUT_ARG_NOT_WELL_FORMED_ERR')
```

Comment:
This is a change in behavior. Users relying on the error message output instead of the code returned to the shell will need to adjust their scripts.

Reply:
The change in the error code is acceptable because the parser will be shipped in iRODS 5 only.

However, it should be documented in the changelog / release notes.

Will open an issue for this so we don't forget it.

Reply:
Created #8096.